### ISSUE TITLE
Doesn't work on Colab

### ISSUE THREAD

**Issue by aced125** [facebookresearch/BLINK#42]
Getting this error when trying to run it: tcmalloc: large alloc 24180850688 bytes

Have followed all instructions.

**Comment by ledw** [comment]
@aced125 Hi, the model requires a large amount of memory to run. The issue seems to be complaining that there's isn't enough memory available.

**Comment by YerongLi** [comment]
Yeah, this model requires a large RAM to run, according to this: #2, does anyone finds a workaround.
